core/templates/newspapers.html

```diff
@@ -48,7 +48,7 @@
                     {{ place.get_city_county }}<br/>
                 {% endfor %}
                 </td>
-                <td><a href="{% url 'openoni_title' lccn %}"><strong>{{title.display_name}}</strong></a><br />{{title.place_of_publication}}, {{title.start_year}}-{{title.end_year}}</td>
+                <td scope="row"><a href="{% url 'openoni_title' lccn %}"><strong>{{title.display_name}}</strong></a><br />{{title.place_of_publication}}, {{title.start_year}}-{{title.end_year}}</td>
```

Comment:
I was unfamiliar with these so I looked them up. Looks like this attribute is obsolete for `<td>` elements:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td